build-environment.yml

```diff
@@ -4,6 +4,8 @@ channels:
 dependencies:
 - python =3.12
 - mamba
+- micromamba =2.0.5
+- nodejs =22
```

Comment:
It would be great to not have to depend on `nodejs` as a user of `jupyterlite-terminal`. 

But this can likely be tracked in https://github.com/jupyterlite/terminal or https://github.com/jupyterlite/cockle.

Reply:
Removing it is not an option,